html/base/list.stories.js

```diff
@@ -0,0 +1,68 @@
+export default {
+  title: 'Components|List',
+  decorators: [
+    story => `<div class="sprk-o-Box">${story()}</div>`,
+  ],
+};
+
+export const unordered = () => (
+  `
+    <ul
+      class="sprk-b-List"
+      data-id="unordered-list-1"
+    >
+      <li data-id="list-item-1">Unordered List Item</li>
+      <li data-id="list-item-2">Unordered List Item</li>
+      <li data-id="list-item-3">Unordered List Item</li>
+    </ul>
+  `
+);
+
+export const ordered = () => (
+  `
+    <ol
+      class="sprk-b-List"
+      data-id="ordered-list-1"
+    >
+      <li data-id="list-item-4">Ordered List Item</li>
+      <li data-id="list-item-5">Ordered List Item</li>
+      <li data-id="list-item-6">Ordered List Item</li>
+    </ol>
+  `
+);
+
+export const indented = () => (
+  `
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      Sed gravida urna quis nulla ultrices, sed efficitur
+      risus elementum.
+    </p>
```

Comment:
Same thing about the paragraphs here with them not being part of the list component.